cm-mlops/script/convert-ml-model-huggingface-to-onnx/customize.py

```diff
@@ -0,0 +1,28 @@
+from cmind import utils
+import os
+
+def preprocess(i):
+
+    os_info = i['os_info']
+
+    env = i['env']
+
+    if env.get("CM_MODEL_HUGG_PATH","") == "":
+        return {'return': 1, 'error': 'CM_MODEL_HUGG_PATH is not set'}
+    
+    automation = i['automation']
+
+    cm = automation.cmind
+
+    path = os.getcwd()
+
+    return {'return':0}
+
+def postprocess(i):
+    os_info = i['os_info']
+
+    env = i['env']
+    env['HUGGINGFACE_ONNX_FILE_PATH'] = os.path.join(os.getcwd(),"model.onnx")
+    with open('tmp-run-env.out', 'w') as f:
```

Comment:
Thank you @anandhu-eng for the change. Can you please remove this line? Unlike native files, inside customize.py we have direct access to env dictionary and so we don't need to use tmp-run-env.out.

Reply:
oh okay sure

Reply:
modified 👍🏼